### ISSUE TITLE
content in a "drafted state" can fail to push publish.

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#21714]
If you have a content object that has both a live and working version in an sender environment and an older version of that object already lives on the receiver (neither the live or working inodes have been pushed), when we push the new version of the object, we can get the error that the live version does not exist on the receiver.

```
Failed to Publish	Failed to publish because an error occurred: An error occurred when processing Contentlet in '/data/shared/assets/bundles/01FVWVAW7HNSDE7SWX02123CNE/working/System Host/1/2-content.0ca99dc0-0d78-424e-88b0-4db6c087925b.content.xml' with ID 'e7246b8e-d3ea-4ef3-a096-45cabf4e0c3a': 
'ERROR: insert or update on table "contentlet_version_info" violates foreign key constraint "fk_contentlet_version_info_live" 
Detail: Key (live_inode)=(e8ce5f49-5f7c-4d76-8f7e-c00c5afccd80) is not present in table "contentlet".
{ "SQL": ["UPDATE contentlet_version_info SET working_inode=?, live_inode=?, deleted=?, locked_by=?, locked_on=?, version_ts=? WHERE identifier=? AND lang=?"], "maxRows": [-1], "offest": [0], "params": [ "777e0f71-f87d-4cdc-8548-498bdc9c453a", "e8ce5f49-5f7c-4d76-8f7e-c00c5afccd80", false, "dotcms.org.5", "2022-02-11 09:59:38.207", "2022-02-14 14:32:00.949", "e7246b8e-d3ea-4ef3-a096-45cabf4e0c3a", 1 ] }'
```

#### Bundle
[01FVWVAW7HNSDE7SWX02123CNE.tar.gz](https://github.com/dotCMS/core/files/8063511/01FVWVAW7HNSDE7SWX02123CNE.tar.gz)

#### Sender
![Screen Shot 2022-02-14 at 2 39 04 PM](https://user-images.githubusercontent.com/934364/153934348-a12dd8d8-fdba-46bc-bac8-95d175129566.png)

#### Receiver
![Screen Shot 2022-02-14 at 2 40 37 PM](https://user-images.githubusercontent.com/934364/153934578-a77f284e-ef16-4549-afb5-3969b5cccf75.png)

**Comment by freddyucv** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Falcon-6176e9d907eb0000106ee8e1/issues/dotCMS/core/21714?planning-poker) with ZenHub @dsilvam @victoralfaro-dotcms

**Comment by dsilvam** [comment]
PR ee: https://github.com/dotCMS/enterprise/pull/853

**Comment by bryanboza** [comment]
Fixed, tested on release-22.04 // Docker // FF
